css/src/components/timeline.scss

```diff
@@ -0,0 +1,51 @@
+$timeline-content-font-size: $font-size-8 !default;
+$timeline-timestamp-font-size: $font-size-9 !default;
+$timeline-timestamp-font-weight: $weight-semilight !default;
+$timeline-description-padding-top: $spacer-4 !default;
+$timeline-description-line-height: $line-height-normal;
+
+$timeline-item-border-left: $border-width solid $border !default;
+$timeline-item-min-height: $layout-6 !default;
+$timeline-item-padding-bottom: $spacer-7 !default;
+$timeline-item-padding-left: $spacer-8 !default;
+
+$timeline-badge-top-offset: -0.35em !default;
+$timeline-badge-left-offset: -1.05em !default;
+
+.timeline-item {
+	display: flex;
+	position: relative;
+	padding-block-end: $timeline-item-padding-bottom;
+	padding-inline-start: $timeline-item-padding-left;
+	min-height: $timeline-item-min-height;
+	border-inline-start: $timeline-item-border-left;
+
+	&:last-child {
+		border-inline-start: none;
+	}
+
+	.timeline-item-badge {
+		position: absolute;
+		inset-block-start: $timeline-badge-top-offset;
+		inset-inline-start: $timeline-badge-left-offset;
+	}
+
+	.timeline-item-content {
+		display: flex;
+		flex-direction: column;
+		flex-grow: 1;
+		font-size: $timeline-content-font-size;
+
+		.timeline-item-timestamp {
```

Comment:
1. Please review the component naming documentation, if this is intended to live within `timeline-item-content`, it should be prefixed with that, resulting in `.timeline-item-content-<subcomponentname>`.
1. Since this does not necessarily only contain the timestamp, but also seems to include the title, I think we should re-consider the name. I think there's a couple ways we could go:
    - First, we could remove these, and achieve the same with atomics only. The `.timeline-item-content` would just become a box. 
    - Second, we could update the `.timeline-item-content` to have a very rigid `display: grid` based structure, which could  -title, -time, -description subcomponents that fit directly into `.timeline-item-content`

Reply:
Fixed via conversation